pom.xml

```
@@ -4,7 +4,7 @@
     <parent>
         <groupId>org.jenkins-ci.plugins</groupId>
         <artifactId>plugin</artifactId>
-        <version>4.40</version>
+        <version>4.73</version>
```

Would be nice (not required, but nice) to use the most recent release, 4.74.
```suggestion
        <version>4.74</version>
```